helm/scylla-manager/templates/manager_serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.controllerServiceAccount.create -}}
```

Comment:
@zimnx would we just always create the SA account with the fixed name? we own the namespace

Reply:
It's a common pattern in Helm (https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L131, https://github.com/bitnami/charts/blob/master/bitnami/cassandra/values.yaml#L60), and it's autogenerated in the chart scaffold. 
Although I'm not enterly sure what's the usecase for disabling creation of serviceAccount. Perhaps adding some custom stuff like image secrets, but these could be added to the SA template.
I'm fine with always creating it, but that would be a regression compared to 1.1.